docs/user-guide/commands/argocd_app_delete.md

```diff
@@ -14,6 +14,7 @@ argocd app delete APPNAME [flags]
 
 ```
       --cascade   Perform a cascaded deletion of all application resources (default true)
+      --force     Turn off prompting to confirm cascaded deletion of application resources
```

Comment:
I think we should reserve the word `--force` for something else. (e.g. is there a `kubectl delete --force` that this should match up with instead?

Reply:
Yeah, this is odd.

Why can't the `--cascade` flag change to `default=false`?